ext/MitsubaLoader/CMitsubaLoader.h

```diff
@@ -18,6 +20,50 @@ namespace MitsubaLoader
 
 namespace bsdf
 {
+#include "irr/irrpack.h"
+	//must be 64bit
+	struct STextureData
```

Comment:
this should 100% be part of the ITexturePacker

Reply:
done